src/cli/cli.rs

```diff
@@ -306,6 +306,24 @@ pub fn build_cli() -> App<'static, 'static> {
                 )
                 .arg(color_arg.clone()),
         )
+        .subcommand(
+            SubCommand::with_name("colorblind")
+                .about("Simulate a color under a certain colorblindness profile")
+                .long_about(
+                    "Convert the given color to how it would look to a person with protanopia, \
+                    deuteranopia, or tritanopia \n\n\
+                     Example:\n  \
+                       pastel random | pastel colorblind deut")
```

Comment:
minor point: using input from `pastel random` is maybe not the most interesting use case, because you wouldn't know how the color looked like before. Maybe `pastel distinct 3 | pastel colorblind deut`?